src/azure-cli/azure/cli/command_modules/cloud/__init__.py

```diff
@@ -40,7 +40,7 @@ def load_arguments(self, command):
         # pylint: disable=line-too-long
         with self.argument_context('cloud') as c:
             c.argument('cloud_name', options_list=('--name', '-n'), help='Name of a registered cloud', completer=get_cloud_name_completion_list)
-            c.argument('profile', help='Profile to use for this cloud', choices=list(API_PROFILES))
+            c.argument('profile', help='Profile to use for this cloud. The azure stack profiles `2017-03-09-profile` `2018-03-01-hybrid` `2019-03-01-hybrid` and `2020-09-01-hybrid` have been deprecated and removed. To continue using Azure Stack please install the CLI 2.66.* (LTS) version.', choices=list(API_PROFILES))
```

Comment:
Minor catch: Shall we add quote for 2.66.* to align Line 607 or stripe both off?